shell/app/electron_library_main.mm

```diff
@@ -37,8 +38,7 @@ int ElectronMain(int argc, char* argv[]) {
 
 int ElectronInitializeICUandStartNode(int argc, char* argv[]) {
   if (!electron::fuses::IsRunAsNodeEnabled()) {
-    CHECK(false) << "run_as_node fuse is disabled";
-    return 1;
+    NOTREACHED() << "run_as_node fuse is disabled";
```

Comment:
I could be wrong about this, but is this going to throw a hard crash if an app has the RunAsNode fuse disabled? We were returning `return 1` here previously if the CHECK failed, I just want to make sure this isn't a behavior change

Reply:
Good catch, I don't think there is a behavior change here. NOTREACHED without a milestone and CHECK use the same underlying destructor `~CheckError` https://source.chromium.org/chromium/chromium/src/+/main:base/check.h which will abort, the return wouldn't be needed previously as well.